### PR TITLE
New opt-out page design

### PR DESCRIPTION
**Pull request by debugwand**
screen shot 2016-09-30 at 17 56 29

### CODE REVIEW

File: client/js/validate-feedback.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+const Validator = {
+	init: function (form) {
+		this.form = form;
+		this.getUserInputs();
+		this.eventListeners();
+	},
+
+	getUserInputs: function () {
+		this.textareas = [].slice.call(this.form.querySelectorAll('textarea'));
+		this.checkboxes = [].slice.call(this.form.querySelectorAll('input[type="checkbox"]'));
```
Comment (phamann) [inline client/js/validate-feedback.js]: FYI we transpile the client-side JS, so could use `...` spread here instead.

File: client/js/validate-feedback.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		}
+	},
+
+	hasUserEntry: function () {
+		const hasChecked = this.checkboxes.some(i => {
+			return i.checked;
+		});
+		const hasText = this.textareas.some(t => {
+			return t.value.length;
+		});
```
Comment (phamann) [inline client/js/validate-feedback.js]: Both of these can be condensed to single lines.

File: client/main.scss [hunk 3]
```diff
@@ -13,8 +16,68 @@ $output-critical: true !default;
...
 
+/* new confirm page */
+
+.optout__heading {
+	@include oTypographyHeading2();
+}
+
+.optout-strapline {
+	@include oTypographyBody();
+	@include font-size(16, 19);
```
Comment (phamann) [inline client/main.scss]: Why is this off the type scale?

Comment (debugwand) [inline client/main.scss]: body is 18px, and as far as I can see doesn't come with modifiers, 18px was too large (went through design review with Patricia)

Comment (debugwand) [inline client/main.scss]: The sizing was taken from the serif 's' sizes set out here: https://github.com/Financial-Times/o-typography#readme. After a quick talk with Patricia, she's said Financier is fine

**Pull request merged**